app/models/cdo/create/address.js

```diff
@@ -46,34 +53,32 @@ function ViewModel (address, errors) {
       autocomplete: forms.preventAutocomplete,
       attributes: { maxlength: '8' }
     },
-    country: {
-      id: 'country',
-      name: 'country',
-      label: {
-        text: 'Country'
-      },
-      value: address.country,
-      autocomplete: forms.preventAutocomplete,
-      attributes: { maxlength: '30' }
-    },
+    countries: mapToCountrySelector(countries, validationError, address.country),
     errors: []
   }
 
-  if (errors) {
-    for (const error of errors.details) {
+  if (validationError) {
+    for (const error of validationError.details) {
       const name = error.path[0]
 
-      const prop = this.model[name]
-
-      if (prop !== undefined) {
-        prop.errorMessage = {
-          text: error.message
-        }
-
+      if (name === 'country') {
```

Comment:
Not sure why you have to deal with 'county' explicitly. Does is not follow the generic error flow?

Reply:
the model.country change resolves this perfectly.  Thanks again.